controllers/highlightController.js

```diff
@@ -0,0 +1,232 @@
+import joi from "joi";
+import models from "../models";
+import constants from "../helpers/constants";
+import errorMessages from "../helpers/errorMessages";
+import {
+  highlightValidator,
+  editHighlightValidator,
+  oneHighlightValidator
+} from "../helpers/highlightValidator";
+
+const { Highlight } = models;
+const {
+  OK,
+  INTERNAL_SERVER_ERROR,
+  BAD_REQUEST,
+  CREATED,
+  NOT_FOUND
+} = constants.statusCode;
+const { INTERNAL_SERVER_ERROR_MESSAGE } = errorMessages;
+
+/**
+ * @class HighlightController
+ */
+class HighlightController {
+  /**
+   * @description Highlight a given article
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @param  {Object} next - The next middleware.
+   * @return {Object} - It returns the response object.
+   */
+
+  static create(req, res, next) {
+    const { slug } = req.params;
+    const { start, end } = req.query;
+    const { comment } = req.body;
+    const { article } = req;
+    const { id: userId } = req.user;
+    // eslint-disable-next-line no-unused-vars
```

Comment:
@espoirMur Talk to your team to see if you need to disable this rule globally, otherwise, you can not disable every rule to fit your need.